### PR TITLE
Updated Example in Class:LoraModel

### PR DESCRIPTION
**Pull request by TianyiPeng**
The previous example is no longer functional, but I have made slight modifications to ensure its compatibility with the latest version, making it executable once again.

### CODE REVIEW

**Review by younesbelkada — COMMENTED** [review]
Hi @TianyiPeng 
Thanks for the PR, looking at other model cards docstrings I think this is intended: https://github.com/huggingface/peft/blob/c46d76ae3a323fa208bd777b9eb60a383e094d49/src/peft/tuners/p_tuning.py#L91 
However there is indeed a typo in the current docstring of `LoraModel`, the argument `adapter_name` is missing. https://github.com/huggingface/peft/blob/c46d76ae3a323fa208bd777b9eb60a383e094d49/src/peft/tuners/lora.py#L174
Can you add that in the docstring instead? Thanks!

**Comment by TianyiPeng** [comment]
> Hi @TianyiPeng Thanks for the PR, looking at other model cards docstrings I think this is intended:
> 
> https://github.com/huggingface/peft/blob/c46d76ae3a323fa208bd777b9eb60a383e094d49/src/peft/tuners/p_tuning.py#L91
> 
> 
> However there is indeed a typo in the current docstring of `LoraModel`, the argument `adapter_name` is missing.
> https://github.com/huggingface/peft/blob/c46d76ae3a323fa208bd777b9eb60a383e094d49/src/peft/tuners/lora.py#L174
> 
> 
> Can you add that in the docstring instead? Thanks!

I have a question. Seems that not only the `adapter_name` is missing, we also need that `config[adapter_name]` to be the real `config` (see below). Should we still just change the docstring or maybe we should change the code to add a case for `if adapter_name is None`? Any suggestion?
https://github.com/huggingface/peft/blob/5a0e19dda1048ff8caaa12970ba7574f9cdfbf76/src/peft/tuners/lora.py#L175-L181

I find the same issue for docstrings in other adapters such as: 

https://github.com/huggingface/peft/blob/5a0e19dda1048ff8caaa12970ba7574f9cdfbf76/src/peft/tuners/adalora.py#L78-L84

**Review by younesbelkada — COMMENTED** [review]
Thanks for iterating, I think you are right, I think we should first update the docstring to reflect the current situation then we can think of a potential fix; would you be happy to address these changes on docstrings?

**Comment by TianyiPeng** [comment]
> Thanks for iterating, I think you are right, I think we should first update the docstring to reflect the current situation then we can think of a potential fix; would you be happy to address these changes on docstrings?

Hi, I updated the docstring accordingly. Happy to help more if needed.

File: src/peft/tuners/adalora.py [hunk 1]
```diff
@@ -80,7 +80,8 @@ class AdaLoraModel(LoraModel):
                 peft_type="ADALORA", task_type="SEQ_2_SEQ_LM", r=8, lora_alpha=32, target_modules=["q", "v"],
                 lora_dropout=0.01,
             )
-        >>> model = AutoModelForSeq2SeqLM.from_pretrained("t5-base") >>> model = AdaLoraModel(config, model)
+        >>> model = AutoModelForSeq2SeqLM.from_pretrained("t5-base") 
+        >>> model = AdaLoraModel(model, config, "AdaLoraModel")
```
Comment (younesbelkada) [inline src/peft/tuners/adalora.py:84]:
```suggestion
        >>> model = AdaLoraModel(model, config, "default")
```

File: src/peft/tuners/lora.py [hunk 2]
```diff
@@ -136,7 +135,7 @@ class LoraModel(torch.nn.Module):
         ... )
 
         >>> model = AutoModelForSeq2SeqLM.from_pretrained("t5-base")
-        >>> lora_model = LoraModel(config, model)
+        >>> lora_model = LoraModel(model, config, "LoraModel")
```
Comment (younesbelkada) [inline src/peft/tuners/lora.py:138]:
```suggestion
        >>> lora_model = LoraModel(model, config, "default")
```

**Review by younesbelkada — COMMENTED** [review]
Thank you very much for iterating , I left two comments, we use `"default"` as the default adapter name! apart from that LGTM !

**Comment by TianyiPeng** [comment]
> Reference

Done!

**Review by younesbelkada — COMMENTED** [review]
Thanks a lot for iterating, can you run the styling checks?

```bash
make style && make quality
```
Then we should be able to merge I think !

**Review by younesbelkada — APPROVED** [review]
Thanks a lot for this!

**Pull request merged**